### PR TITLE
feat: add CORS proxy for Google calendar

### PR DESCRIPTION
**Pull request by LeonVreling**

### CODE REVIEW

**Comment by serge1peshcoff** [comment]
@linuxbandit calendar.google.com is being bitchy about cors, therefore this

**Review by serge1peshcoff — APPROVED** [review]

File: oms-global/docker/traefik/traefik.toml.dev [hunk 1]
```diff
@@ -94,7 +94,17 @@ swarmmode = false
...
+  [frontends.calendar-cors-frontend]
+  backend = "calendar-cors-backend"
+  priority = 20
+  [frontends.calendar-cors-frontend.routes.main]
+    rule = "PathPrefixStrip:/services/calendar-cors/;"
+
 [backends]
   [backends.jira-cors-backend]
     [backends.jira-cors-backend.servers.server1]
     url = "https://oms-project.atlassian.net/"
```
Comment (linuxbandit) [inline oms-global/docker/traefik/traefik.toml.dev:106]: @serge1peshcoff / @LeonVreling one of you could have updated this one :D

Comment (serge1peshcoff) [reply oms-global/docker/traefik/traefik.toml.dev:106]: ah crap, indeed. This shouldn't be critical as we have redirect there right?

Comment (linuxbandit) [reply oms-global/docker/traefik/traefik.toml.dev:106]: yes idk for how long (any organisation could claim that name, and redirect would break of course)

Comment (serge1peshcoff) [reply oms-global/docker/traefik/traefik.toml.dev:106]: @LeonVreling can you make a PR and update it? by replacing oms-project with myaegee in URL for jira

**Pull request merged**